test/cache/cache-test.c

```diff
@@ -77,7 +77,7 @@ main() {
     it("should manage the search cache") {
       char *cached_search;
 
-  clib_cache_delete_search();
+      clib_cache_delete_search();
```

Comment:
Ah whoops, #209 should have formatted tests too. I forgot we have `.c` tests now :)

Reply:
#211 :)